src/org/labkey/test/components/ui/files/FileAttachmentContainer.java

```diff
@@ -0,0 +1,249 @@
+package org.labkey.test.components.ui.files;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.components.html.FileInput;
+import org.labkey.test.components.html.Input;
+import org.openqa.selenium.ElementNotInteractableException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.io.File;
+import java.util.List;
+
+/*
+    Wraps https://github.com/LabKey/labkey-ui-components/blob/fb_assayResultsFiles/packages/components/src/internal/components/files/FileAttachmentContainer.tsx
```

Comment:
Don't use the feature branch in this url
```suggestion
    Wraps https://github.com/LabKey/labkey-ui-components/blob/develop/packages/components/src/internal/components/files/FileAttachmentContainer.tsx
```

Reply:
omg thanks nice catch